icu4c/source/i18n/unicode/numberrangeformatter.h

```diff
@@ -185,8 +185,14 @@ class NumberRangeFormatterImpl;
  * Export an explicit template instantiation. See datefmt.h
  * (When building DLLs for Windows this is required.)
  */
-#if U_PF_WINDOWS <= U_PLATFORM && U_PLATFORM < U_PF_CYGWIN && !defined(U_IN_DOXYGEN)
-template struct U_I18N_API std::atomic<impl::NumberRangeFormatterImpl*>;
+#if U_PF_WINDOWS <= U_PLATFORM && U_PLATFORM <= U_PF_CYGWIN && !defined(U_IN_DOXYGEN)
+} // namespace icu::number
+U_NAMESPACE_END
+
+template struct U_I18N_API std::atomic<icu::number::impl::NumberRangeFormatterImpl*>;
```

Comment:
Same as above, possibly add <>.

Reply:
When I add them here the Cygwin-x64/GCC build fails.

Reply:
Hmm, then leave them out, I guess.

Reply:
I added them to the `umutex.h` file, but will leave them out of `numberrangeformatter.h`.